swift/actions/build-and-test/action.yml

```diff
@@ -44,10 +44,10 @@ runs:
         mkdir -p bazel-cache/{repository,disk}
         echo build --repository_cache=bazel-cache/repository --disk_cache=bazel-cache/disk > local.bazelrc
         echo test --test_output=errors >> local.bazelrc
-    - name: Print unextracted entities
-      shell: bash
-      run: |
-        bazel run //swift/extractor/print_unextracted
+    # - name: Print unextracted entities
+    #   shell: bash
+    #   run: |
+    #     bazel run //swift/extractor/print_unextracted
```

Comment:
This is only used internally, and now it started failing due to some missing dylibs. We'll fix it later.